config/locales/en.yml

```diff
@@ -20,4 +20,160 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  full_title: "Open Source Friday | This Friday, invest a few hours contributing to the software you use and love."
```

Comment:
I reckon split this into two strings and make use the the `Open Source Friday` string immediately below.